content/docs/practices/the_zen.md

```diff
@@ -0,0 +1,118 @@
+---
+title: The Zen of Prometheus
+sort_rank: 9
+---
+
+# The Zen of Prometheus
+
+**The Zen of Prometheus** is beginner friendly set of core values and guidelines for instrumenting your applications and writing idiomatic alerts using Prometheus.
+This is a document that's intended to maintained by the Prometheus community. Feel free to [contribute](https://github.com/prometheus/docs).
+
+## Instrument first, ask questions later
+
+During development you will never know what questions you need to ask later. Software needs good instrumentation, it's not optional. Metrics are cheap. Use them generously.
+
+The First and the most important rule, if you have to remember only one thing remember this one. Instrument all the things!
+
+## Measure what users care
+
+Do your users care if your database servers are down? Do they care about your CPU saturation? No, they care about what they experience. They care about whether they can access the page that they have requested and their results are fresh. Think in terms of latencies and availability. Let your [SLO](https://landing.google.com/sre/sre-book/chapters/service-level-objectives/)s guide your instrumentation.
```

Comment:
Uhm, you should be measuring all of those things.

You're conflating metrics with alerting here.

Reply:
I think this should be phrased in a way that measuring all of those things is still important, but that instrumentation should _additionally_ be suitable for SLO-based alerting.